core/metacling/src/TCling.cxx

```diff
@@ -2442,7 +2442,7 @@ static int HandleInterpreterException(cling::MetaProcessor* metaProcessor,
    }
    catch (cling::InterpreterException& ex)
    {
-      Error("HandleInterpreterException", "%s.\n%s", ex.what(), "Execution of your code was aborted.");
+      Error("HandleInterpreterException", "%s\n%s", ex.what(), "Execution of your code was aborted.");
```

Comment:
Perhaps we should add a missing `.` in `Interpreter.cpp:1845` for projectroot.roottest.cling.specialobj.roottest_cling_specialobj_runf02 not to fail.

Reply:
I'd rather update the ref file: adding a `.` after an expression printout seems "fragile", say `a+1..` is really hard to understand